packages/metronome/build

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+pushd "/pkg/src/metronome"
+rm -rf bin/metronome.bat README.md RUNNING_PID share/ logs/
```

Comment:
Metronome is new, why are things like RUNNING_PID inside the package? That seems like something we should fix in the upstream.